examples/react-ts/.storybook/main.ts

```diff
@@ -22,7 +22,7 @@ const config: StorybookConfig = {
     },
   },
   core: {
-    builder: 'webpack4',
+    builder: { name: 'webpack4' },
```

Comment:
I guess this change is not necessary, unless you want to

Reply:
It's a test